src/In.ProjectEKA.HipService/DataFlow/DataFlow.cs

```diff
@@ -53,6 +55,55 @@ public class DataFlow : IDataFlow
             return new Tuple<HealthInformationResponse, ErrorRepresentation>(response, errorRepresentation);
         }
 
+        private static Tuple<HealthInformationResponse, ErrorRepresentation> ConsentArtefactNotFound()
+        {
+            return new Tuple<HealthInformationResponse, ErrorRepresentation>(null,
+                new ErrorRepresentation(new Error(ErrorCode.ContextArtefactIdNotFound,
+                    ErrorMessage.ContextArtefactIdNotFound)));
```

Comment:
> new Error(ErrorCode.ContextArtefactIdNotFound,
                    ErrorMessage.ContextArtefactIdNotFound))
This could be a static field in Error class or some other class and you can reuse it without defining one more method here.

Same for all the cases.